CommandsManual.md

```diff
@@ -10,6 +10,7 @@
 **[impt auth login](#auth-login)**<br />
```

Comment:
You may add a general note to the top of this file, something about how some of the internals of impt have been updated to support multiple DGs, and while we have added some new features, other features have not been re-tested in full so YMMV (YMMV = your-mileage-may-vary).

Also, I'd add the same disclaimer to the README.md file so it's front and center.